sdk/keyvault/keyvault-admin/src/generated/models/mappers.ts

```diff
@@ -104,7 +104,7 @@ export const Permission: coreHttp.CompositeMapper = {
           element: { type: { name: "String" } }
         }
       },
-      notActions: {
```

Comment:
Oops! Not supposed to change this file 😄 

I'm putting up a fix for this shortly